src/components/TrackSelection.tsx

```diff
@@ -1,44 +1,27 @@
-import React, { useCallback } from 'react'
+import React from 'react'
+import { Link } from 'react-router-dom'
 
-import TRACKS from '../data/tracks.json'
+import TRACKS_DATA from '../data/tracks.json'
 
-import { useTrack } from '../hooks/useUrlState'
+const TRACKS = TRACKS_DATA as ReadonlyArray<TrackData>
 
-const ENABLED_TRACKS = TRACKS as ReadonlyArray<TrackData>
-
-function TrackSelectionItem({
-  track,
-  onSelect,
-}: {
+interface TrackSelectionItemProps {
   track: TrackData
-  onSelect: () => void
-}): JSX.Element {
+}
+
+function TrackSelectionItem({ track }: TrackSelectionItemProps): JSX.Element {
   return (
     <li className="list-inline-item mb-2">
-      <button className={`btn btn-md btn-outline-primary`} onClick={onSelect}>
-        {track.name}
-      </button>
+      <Link to={`/${track.slug}`}>
+        <button className={`btn btn-md btn-outline-primary`}>
+          {track.name}
+        </button>
+      </Link>
```

Comment:
Don't nest a `button` inside an `a`chor, as this will result in 2 tab targets, where the inner one doesn't work (try out with tabbing).

Instead:


```suggestion
      <Link to={`/${track.slug}`} className={`btn btn-md btn-outline-primary`}>
        {track.name}
      </Link>
```